src/app/components/MessageBanner/index.tsx

```diff
@@ -12,64 +10,59 @@ import styles from './index.styles';
 import { ServiceContext } from '../../contexts/ServiceContext';
 
 interface MessageBannerProps {
-  summaries: Summary[];
-  title: string;
-  position: number;
+  heading: string;
+  description: string;
+  link: string;
+  linkText: string;
+  image: string;
```

Comment:
Does `image` need to be optional? i.e `image?`. I can't recall whether it was finally decided that there may be no image on a banner

Reply:
Even though the designs had the image as optional, we realised that the data always had an image. Aaron had a similar comment, which I replied to already - https://github.com/bbc/simorgh/pull/10581#discussion_r1099840177 - but the TLDR was, let's leave it as required for now, and next time the message banner is required on a non-Topic page, then we can think about making it optional, and adding the appropriate tests then.